libraries/AP_BoardConfig/AP_BoardConfig.h

```diff
@@ -189,6 +189,9 @@ class AP_BoardConfig {
 
 #if HAL_HAVE_IMU_HEATER
     void set_imu_temp(float current_temp_c);
+    float get_heater_duty_cycle(void) {
```

Comment:
as mentioned on the dev call, let's add a comment to clarify the units of the method.